pkg/apis/aws/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -58,7 +58,7 @@ func (in *AccountClaim) DeepCopyInto(out *AccountClaim) {
 	out.TypeMeta = in.TypeMeta
 	in.ObjectMeta.DeepCopyInto(&out.ObjectMeta)
 	in.Spec.DeepCopyInto(&out.Spec)
-	out.Status = in.Status
+	in.Status.DeepCopyInto(&out.Status)
```

Comment:
@jharrington22 Still I dont understand why are you using `Conditions`? How will UHC get the `AccountClaim` *status* to verify that the account is `Ready`? What should UHC expect the status field of the API to look like?

Reply:
@nimrodshn UHC would watch the conditions where .status.conditions[].type would be Claimed and the .status.conditions.status=true

Reply:
Which condition - `Conditions[0]` ? 
What does `Conditions[1:]` mean?
Im not sure I understood - the account should NOT be claimed as UHC is the one doing the claiming.

cc: @jhernand

Reply:
@jharrington22 Just for clarity I would iterate over Conditions like the following:
```
for _, condition := range Conditions {
    if condition.type == "Claimed" && condition.Status == corev1.ConditionTrue {
    // expect there be valid AWS credentials in spec.AWS
    }
}
```
Is that correct?

Reply:
@nimrodshn after speaking with Matt and Naveen yesterday we are going to add a status field as well as the conditions on a claim to simplify things. This will also help for anyone looking at a claim for the first time as they wouldn't have to sort through the conditions.

As for your comment yes that is how I would expect someone to iterate over conditions. I'd be holding off with this PR until the status discussion was closed I will get this merged today.